### PR TITLE
Add Community information

### PR DESCRIPTION
**Pull request by povilasv**

### CODE REVIEW

File: docs/community.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+slug: /community.md
+---
+
+# General
+
+Thanos is an open source project and we value and welcome new contributors and members of the community. Here are ways to get in touch with the community:
+
+## Thanos Community meeting
+
+Thanos Community Meeting is a PUBLIC and RECORDED monthly meeting every second Wednesday 12:00 UTC ([Time zone converter](https://www.thetimezoneconverter.com/?t=12%3A00%20pm&tz=UTC%20(Coordinated%20Universal%20Time)&)).
```
Comment (bwplotka) [inline docs/community.md:15]:
```suggestion
Thanos Community Meeting is a **public** and **recorded** monthly meeting every second Wednesday 12:00 UTC ([Time zone converter](https://www.thetimezoneconverter.com/?t=12%3A00%20pm&tz=UTC%20(Coordinated%20Universal%20Time)&)).
```

File: docs/community.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: Community
+type: docs
+menu: thanos
```
Comment (bwplotka) [inline docs/community.md:4]: maybe `contributing` instead? main menu is bloated already

**Comment by bwplotka** [comment]
@brancz we need your "friendly eye" on this one! (: 

(you are the friendlies human on the earth)

**Comment by s-urbaniak** [comment]
amazing! :+1:

**Comment by povilasv** [comment]
We've added a link to https://calendar.google.com/calendar/embed?src=go39q7eu71vvu3gfrkbup6b254%40group.calendar.google.com you can click on it and copy to calendar

**Comment by brancz** [comment]
ok I guess that's good enough for now. maybe in the future we can have a google group that auto-invites you if you join

**Comment by povilasv** [comment]
Cool I need to learn how to do that, so we can do the same thing for Prometheus Community meeting ;)

File: docs/community.md (hunk 2 above)
Comment (povilasv) [reply docs/community.md]: I've moved under contributing :)

**Pull request merged**